CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+The command `go get "github.com/getyoti/yoti-go-sdk"` downloads the Yoti package, along with its dependencies, and installs it.
+
+## Commit Process
+
+1) `go fmt` to format the code
+1) `go vet` reports suspicious constructs
+1) `go test` to run the tests
+
+## VS Code
+
+For developing in VS code, use the following `launch.json` file (placed inside a `.vscode` folder) to easily run the examples from VS Code:
+
+```json
```

Comment:
Because you're using comments, it's probably better to set the syntax to `javascript`. Seems like the comments look all read otherwise: https://github.com/getyoti/yoti-go-sdk/blob/SDK-251-AML/CONTRIBUTING.md